javascripts/discourse/lib/isValidUrl.js

```diff
@@ -1,6 +1,12 @@
 export default function isValidUrl(string) {
+  let url;
+
   try {
-    URL(string);
+    url = new URL(string);
```

Comment:
Instead of having `let` above, I think we can have const like this:
```
const url = new URL(string);
```

Reply:
@lis2 Thanks, good point, I've pushed an update.